search/tests/test_indexing.py

```diff
@@ -101,10 +110,15 @@ def test_sorting(self):
         )
         # expected position (res2, res3, res1)
         top_res = results[0]
+        sec_res = results[1]
         self.assertEqual(
             top_res.nr_views,
             res2.xa_nr_views
         )
+        self.assertEqual(
```

Comment:
If you change this:

```python
        self.assertEqual(
            top_res.nr_views,
            res2.xa_nr_views
        )
        self.assertEqual(
            sec_res.nr_views,
            res4.xa_nr_views
        )
```

to this:

```python
        self.assertEqual(
            top_res.nr_views,
            res4.xa_nr_views
        )
        self.assertEqual(
            sec_res.nr_views,
            res2.xa_nr_views
        )
```

You will get the same results (the tests will still pass), but it won't be correct.